assets/src/components/vehiclePropertiesCard.tsx

```diff
@@ -84,20 +92,24 @@ const TrNameValue = ({
   )
 }
 
-const VehicleWorkInfo = ({ vehicle }: VehicleProp): React.ReactElement => (
+const VehicleWorkInfo = ({
+  vehicle,
+}: VehicleOrGhostProp): React.ReactElement => (
   <>
     <table className="m-vehicle-work-info">
       <tbody className="m-vehicle-work-info__items">
-        <TrNameValue name="run">{vehicle.runId ?? "N/A"}</TrNameValue>
-        <TrNameValue name="vehicle">{vehicle.label ?? "N/A"}</TrNameValue>
+        <TrNameValue name="run">{vehicle.runId || "N/A"}</TrNameValue>
+        <TrNameValue name="vehicle">
+          {(isVehicle(vehicle) && vehicle.label) || "N/A"}
+        </TrNameValue>
         <TrNameValue name="operator" sensitivity={HideSensitiveInfo.All}>
-          <span className="title-case">
-            {joinTruthy([
+          {(isVehicle(vehicle) &&
+            joinTruthy([
               vehicle.operatorFirstName,
               vehicle.operatorLastName,
               vehicle.operatorId && `#${vehicle.operatorId}`,
-            ]).toLowerCase() || "N/A"}
-          </span>
+            ])) ||
+            "N/A"}
         </TrNameValue>
       </tbody>
```

Comment:
I considered leaving the span and changing the class, but it felt like it was a bit useless to leave it (aside that it changed snapshots)